src/modules/register/Register.jsx

```diff
@@ -252,210 +240,124 @@ class RegisterForm extends React.Component {
 	constructor(props) {
 		super(props);
 
-		this.inputs = {
-			name: React.createRef(),
-		  email: React.createRef(),
-		  password: React.createRef(),
-		}
+		this.SCREENS = {
+			NAME: 'name',
+			PASSWORD: 'password',
+			EMAIL: 'email'
+		};
```

Comment:
This is just to give variable names to the screens, so that we don't have a bunch of strings (`'name', 'password', 'email'`) floating around our component.

This makes it easier to change the name later if we need.